skopt/utils.py

```diff
@@ -0,0 +1,87 @@
+from time import time
+
+import numpy as np
+from scipy.optimize import OptimizeResult
+
+
+def create_result(Xi, yi, space=None, rng=None, specs=None, models=None):
```

Comment:
should we represent the absence of models by `None` or an empty list in `OptimizeResult`? I've seen `[]` somewhere else in the code but I think `None` is a better choice.

What ever we decide, the choice should be checked for in `check_minimizer_api` int `test_common.py`